### ISSUE TITLE
Unable To Connect To The Remote Server

### ISSUE THREAD

**Issue by MrGhost101** [mrpond/BlockTheSpot#498]
Hi, I am getting this error when I run both the blockthespot and blockthespot+spicetify bat files

![image](https://github.com/mrpond/BlockTheSpot/assets/71812549/fa9ff082-9410-495e-9c30-7fd24d3202f3)

And this one when I tried running the automated script in Powershell

![image](https://github.com/mrpond/BlockTheSpot/assets/71812549/26564510-b73d-4930-a63f-f66ff6910273)

I have tried doing almost everything mentioned in the known issues section and additional notes section but nothing seems to work. I have never had this issue with installing Blockthespot before.

Any help would be much appreciated.

Thank You

**Comment by Nuzair46** [comment]
Install manually. its just github being blocked in your region. happens sometimes

**Comment by MrGhost101** [comment]
You mean downloading chrome_elf.zip and copying dpapi.dll and config.ini to spotify folder in appdata? I forgot to mention in the issue that I tried that too but nothing happens. Tried closing and reopening spot but it doesn’t seem to work ☹ Sent from Mail for Windows From: NuzairSent: 30 October 2023 13:15To: mrpond/BlockTheSpotCc: Sagar; AuthorSubject: Re: [mrpond/BlockTheSpot] Unable To Connect To The Remote Server (Issue #498) Install manually. its just github being blocked in your region. happens sometimes—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
